src/SUMMARY.md

```diff
@@ -184,6 +184,10 @@
 - [Sanitizers Support](./sanitizers.md)
 - [Debugging support in the Rust compiler](./debugging-support-in-rustc.md)
 
+# General Guides
+
+- [Editions](guides/editions.md)
```

Comment:
I guess we could place this under the "# Contributing to Rust" section like the stability attr guide.

Reply:
Sure! Posted https://github.com/rust-lang/rustc-dev-guide/pull/1838.